client/packages/common/src/authentication/api/hooks/index.ts

```diff
@@ -6,4 +6,6 @@ export * from './useLogin';
 export * from './useGetUserPermissions';
 export * from './useIsCentralServer';
 export * from './useUpdateUser';
-export * from './usePreference';
+export * from './usePreferences';
+
+export const PREFERENCES_QUERY_KEY = 'preferences';
```

Comment:
This was in `system/Manage` - shouldn't be importing from system into common, so moved here, importing into system now 🙏